code/defines/byondtools.dm

```diff
@@ -7,7 +7,7 @@
 /obj/effect/byondtools/changed
 	icon='icons/effects/tile_effects.dmi'
 	icon_state="changed"
-	layer = LIGHTING_LAYER
+	layer = 10
```

Comment:
Is there a comment in the Bay changes as to why it changed?
It looked to me like you have LIGHTING_LAYER defined, but it's been stripped out of a lot of places